tests/basic/pool_function_test.go

```diff
@@ -214,6 +215,30 @@ var _ = Describe("{PoolExpandRejectConcurrent}", func() {
 			}
 		})
 	})
+
+	// test expansion request on a pool while a previous expansion is in progress is rejected
+	It("Expand a pool while a previous expansion is in progress", func() {
+		expandType := api.SdkStoragePool_RESIZE_TYPE_ADD_DISK
+		targetSize := poolToResize.TotalSize/units.GiB + 100
+		err = Inst().V.ExpandPool(poolIDToResize, expandType, targetSize, true)
+		// wait for expansion to start
+		// TODO: this is a hack to wait for expansion to start. The existing WaitForExpansionToStart() risks returning
+		// when the expansion has already completed.
+		time.Sleep(1)
+		// verify pool expansion is in progress
+		isExpandInProgress, expandErr := poolResizeIsInProgress(poolToResize)
+		if expandErr != nil {
+			log.Fatalf("Error checking if pool expansion is in progress: %v", expandErr)
+		}
+		if !isExpandInProgress {
+			log.Warnf("Pool expansion already finished. Skipping test. Please retry with an app that writes more data to testing volumes. ")
+			return
+		}
+		expandResponse := Inst().V.ExpandPoolUsingPxctlCmd(*storageNode, poolToResize.Uuid, expandType, targetSize+100, true)
```

Comment:
I am not aware if there is a limitation the restricts us to use ExpandPool again here. Can you explain why did we have to use ExpandPoolUsingPxctlCmd here?